### DISCUSSION TITLE
Not Working Audio Download - https://equisync.eocinstitute.org/meditation/element/?direct-login

### DISCUSSION THREAD

**Discussion by FelipeDuarteFerreira** [aclap-dev/video-downloadhelper#1018] [Help and Q&A]
### Your question

Hello guys, how are you?

I have the Video Downloader Helper premium license for Microsoft Edge.
I need help, I can't download the audios created on this site: [EquiSync Element (eocinstitute.org)](https://equisync.eocinstitute.org/meditation/element/?direct-login)
The Video Downloader Helper extension does not recognize the media to download. 
![image](https://github.com/user-attachments/assets/670c8cb8-0ca9-41da-b49e-0c6624b70cdb)

Please could you help me?
Bests,
Felipe Duarte

### Addon & CoApp version and details

version: 9.0.2.12
target: microsoft
channel: stable
lang: en-US
coapp: {"found":true,"new_version":false,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19"}
license: {"accepted":true,"email":"<removed>","key":"7ca4794325674180"}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/126.0.0.0 Safari/537.36 Edg/126.0.0.0
{
    "valid_license_message_has_been_displayed": true,
    "used_sidebar_button": true,
    "used_popup_button": true,
    "download_directory": "C:\\VideoDownloadHelper",
    "show_success_notification_for_icognito": true,
    "theme": "dark",
    "view_options": {
        "all_tabs": false,
        "hide_downloaded": false,
        "low_quality": true,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": true,
        "sort_by_status": true,
        "sort_reverse": false
    },
    "use_wide_ui": false,
    "use_legacy_ui": false,
    "use_sidebar": true,
    "open_count_store": 3,
    "successfull_dl": 13
}

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

I'm using Windows 11 Pro 64-bit version 23H2 build 22631.3880, Firefox 128.0 64-bit, licensed VDH 9.0.2.14a1 beta, CoApp 2.0.19.

I clicked the link you posted & it prompted me for a login.  I was going to give up but I decided to shorten your URL.  I was able to visit this page:

https://equisync.eocinstitute.org/meditation/

There's a bunch of media links on that page so I tried a few.  But every one I tried gave me the same results.  The page never seemed to complete loading.  Here's an example:

![#001](https://github.com/user-attachments/assets/8dd25555-0ee7-4412-a0c1-695293ee24ab)

That thing I'm pointing out just throbbed non-stop.  In addition, my cursor turned into a spinner, indicating something was not completing on the page.  Unfortunately, screen captures don't capture the mouse cursor so you can't see that.  There are many things on the page that would be clickable if only the page would complete loading.  I don't know if this is a result of my not having a user ID on the site or some other problem.  But I can't get the page to load properly so I can't help you.

**Comment by Mattmjs** [comment]
@FelipeDuarteFerreira , you should try in Firefox just to see if works or not. If not you might have share the network activity.
It'll need to be attached here as a .zip file. Open the developer tools and click network. This is how it looks in Edge.
I circled the button for recording the network.

![Screenshot 2024-07-18 170801](https://github.com/user-attachments/assets/020288ec-a224-4c51-b5e5-94c1d771d903)
